.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -1122,6 +1122,76 @@ stages:
         baseImage: $(baseImage)
         command: "CheckBuildLogsForErrors"
 
+- stage: profiler_integration_tests_linux
+  condition: and(succeeded(), eq(variables['isScheduledBuild'], 'False'))
+  dependsOn: [package_linux, generate_variables, master_commit_id]
```

Comment:
are these _actually_ dependent on the package step, or can they run have the `build_profiler_linux` step?

Reply:
we have tests that runs with the native loader and tracer. So we need the full package.
(Maybe we could split the tests at some point)